src/policy/policy.h

```diff
@@ -25,8 +25,8 @@ static constexpr unsigned int DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT - 4000};
 static constexpr unsigned int DEFAULT_BLOCK_MIN_TX_FEE{1000};
 /** The maximum weight for transactions we're willing to relay/mine */
 static constexpr unsigned int MAX_STANDARD_TX_WEIGHT{400000};
-/** The minimum non-witness size for transactions we're willing to relay/mine (1 segwit input + 1 P2WPKH output = 82 bytes) */
-static constexpr unsigned int MIN_STANDARD_TX_NONWITNESS_SIZE{82};
+/** The non-witness size for transactions we're unwilling to relay/mine: CVE-2017-12842 */
+static constexpr unsigned int NONSTANDARD_TX_NONWITNESS_SIZE{64};
```

Comment:
micro-nit (since technically there are many nonstandard sizes)
```suggestion
/** In addition to transactions that are too big, do not relay/mine ones that are 64 non-witness bytes exactly: CVE-2017-12842 */
static constexpr unsigned int NONSTANDARD_TX_NONWITNESS_SIZE{64};
```

Reply:
taken, thanks